docs/conf.py

```diff
@@ -73,10 +74,12 @@ def extract_summary(doc, document):
 
 # Configuration for sphinx.ext.intersphinx
 # https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
-#intersphinx_mapping = {
-#    "hail": ("https://hail.is/docs/0.2", None),
-#}
+intersphinx_mapping = {}
 
+if requests.head("https://hail.is/docs/0.2/objects.inv").status_code == 200:
+    intersphinx_mapping["hail"] = ("https://hail.is/docs/0.2", None)
+else:
+    print("Unable to link to Hail docs (cannot access objects.inv)", file=sys.stderr)
```

Comment:
probably a stupid question, for the main gnomad_methods we use logging rather than printing, is there a reason that print should be used here instead of logging?

Reply:
I figured the timestamps and filtering options provided by logging wouldn't be as useful here as in pipelines, since this script runs in under a minute and other output from Sphinx is not timestamped.

Reply:
Sounds good, thank you for explaining!